frontend/src/features/moti/mapSideBar/InventoryTabs.tsx

```diff
@@ -0,0 +1,80 @@
+import clsx from 'classnames';
+import * as React from 'react';
+import { Tab, Tabs } from 'react-bootstrap';
+import styled from 'styled-components';
+
+interface IInventoryTabsProps {
+  PropertyForm: React.ReactNode;
+  disabled?: boolean;
+}
+
+export enum InventoryTabNames {
+  property = 'property',
+  title = 'title',
+  owner = 'owner',
+  value = 'value',
+  history = 'history',
+  notes = 'notes',
+}
+
+const StyledTabs = styled.div`
```

Comment:
MOTI tab styling.

Reply:
Don't change, but probably should be extracted to its own Tab component.  I think Alejandro is working on this.